README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Introduction
+
+My name is Jordi Vasquez and I am from Gainesville, Florida. I am a 4th in the College majoring in political science and also an MA student in the 1 year program in the Committee on International Relations (CIR) (*Joint BA/MA*). 
+
+## Academic Focus
+
+At UChicago I have focused on topics related to security and comparative politics. I am currently writing my MA thesis on U.S. alliance management with relation to Japan, South Korea, Turkey and Greece.
+
+Since Summer 2018, I have also worked as a research assistant on a project on transitional justice after authoritarian regimes. The website for the project can be found here: <https://www.tjdemstabilitylab.com>
```

Comment:
Alternatively

```
Since Summer 2018, I have also worked as a research assistant on [a project on transitional justice after authoritarian regimes](https://www.tjdemstabilitylab.com).
```

Since Summer 2018, I have also worked as a research assistant on [a project on transitional justice after authoritarian regimes](https://www.tjdemstabilitylab.com).

Matches the more common approach of embedding the link in written text